packages/@uppy/core/src/index.js

```diff
@@ -473,6 +474,21 @@ class Uppy {
       }
     }
 
+    // We can't check maxTotalFileSize if the size is unknown.
+    if (maxTotalFileSize && file.data.size != null) {
+      let totalFilesSize = 0
+      totalFilesSize += file.size
```

Comment:
Should we use `file.data.size` here too?

Reply:
Thank you! Modified.